src/database/schema/ChatSchema.ts

```diff
@@ -198,7 +199,7 @@ class ChatSchema implements BaseDbSchema {
       channelId: json?.channel_id,
       userId: json?.message?.user?.id,
       message: json?.message?.text || json?.message?.message,
-      type: json?.message?.type,
+      type: json?.message?.message_type ?? json?.message?.type,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in this line is to handle the case where `message_type` might not be present in `json.message`. This is a good practice as it makes the code more robust against changes in the data structure. However, it's important to ensure that the fallback value (`json?.message?.type`) is always available or else it could lead to unexpected behavior.
```diff
-      type: json?.message?.type,
+      type: json?.message?.message_type ?? json?.message?.type,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->